.github/workflows/build-publish-pr.yml

```diff
@@ -53,6 +54,52 @@ jobs:
           dockerhub_username: ${{ secrets.DOCKERHUB_READONLY_USERNAME }}
           dockerhub_password: ${{ secrets.DOCKERHUB_READONLY_PASSWORD }}
 
+      - name: Get PR labels
+        id: pr-labels
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
+          # Fetch labels using curl
+          RESPONSE=$(curl --silent --fail --header "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
```

Comment:
I think we get `gh` for free right? In which case we can use something like

`gh pr view xxx --json labels` to get a list of labels

Reply:
Nice! Changed that in 6b348334754345b9391d1d1ab22582d548e7bcc3.